src/worker.js

```diff
@@ -70,20 +65,19 @@ function threadAlert() {
   var text = Array.prototype.slice.call(arguments).join(' ');
   postMessage({cmd: 'alert', text: text, threadId: selfThreadId});
 }
-out = threadPrint;
-err = threadPrintErr;
+this.err = threadPrintErr;
 this.alert = threadAlert;
 
 #if WASM
 Module['instantiateWasm'] = function(info, receiveInstance) {
   // Instantiate from the module posted from the main thread.
   // We can just use sync instantiation in the worker.
-  instance = new WebAssembly.Instance(wasmModule, info);
+  const instance = new WebAssembly.Instance(wasmModule, info);
```

Comment:
I don't think we are supposed to use ES6.

Reply:
`const` has been in Firefox/Chrome since 2013, so I thought it would be fine to use. But I'll change it to a `var` is that's required.

Reply:
Can you open a separate issue for that?   I don't actually know what out current policy is or where its documents but it would be good document this.  Presumably people are free to use tools like babel if they want to target really old JS standards so it should be OK to move forward to reasonably modern version.

Reply:
I recall that closure compiler simply rejects your code if you use any ES6 features. There probably is a flag to enable that, but we aren't using it right now.

I also recall @kripken saying that the policy in Emscripten is to avoid ES6 features. I'll let him clarify the policy here.

I personally think that if a browser supports workers, it probably supports `const`...

Reply:
I've changed the const to a var.